lstchain/calib/camera/pedestals.py

```diff
@@ -226,9 +241,9 @@ def calculate_time_results(
 ):
     """Calculate and return the sample time"""
     return {
```

Comment:
Use of astropy units here looks off. Should probably be `to(u.s)` instead of `.value * u.s`